### PR TITLE
Add App Switcher

### PR DESCRIPTION
**Pull request by jyash97**

### CODE REVIEW

File: src/components/AppHeader/AppSwitcher.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+const AppSwitcher = ({
+ apps, currentApp, history, updateCurrentApp,
+}) => {
+	const menu = (
+		<Menu
+			style={{ maxHeight: 250, overflowY: 'scroll' }}
+			onClick={(e) => {
+				const appName = e.key;
+				updateCurrentApp(appName, apps[appName]);
+				history.push(`/app/${appName}`);
```
Comment (metagrover) [inline src/components/AppHeader/AppSwitcher.js]: This causes the app to lose the active view and always redirects the current view to the overview page.

Comment (jyash97) [reply src/components/AppHeader/AppSwitcher.js]: Like if we are on search preview page and when we switch we want the same page to open for other app?
Am I right?

Comment (metagrover) [reply src/components/AppHeader/AppSwitcher.js]: yes

Comment (jyash97) [reply src/components/AppHeader/AppSwitcher.js]: Sounds good.

**Comment by jyash97** [comment]
@metagrover I have done the changes but the tooltip is not showing up, not sure why ,it is visible in the DOM and React elements but displaying on hover.

**Pull request merged**